library/general/uniqueingresshost/template.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8suniqueingresshost
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sUniqueIngressHost
+        listKind: K8sUniqueIngressHostList
+        plural: k8suniqueingresshost
+        singular: k8suniqueingresshost
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package k8suniqueingresshost
+       
+        violation[{"msg": msg}] {
+          input.review.kind.kind == "Ingress"              
```

Comment:
We should be checking the group as well. In this case that means that the groupVersion (aka apiVersion, the field before "Ingress" in `data.inventory`) starts with:

`networking.k8s.io/`

and that `input.review.kind.group` == `networking.k8s.io`